### ISSUE TITLE
./checkra1n: Syntax error: redirection unexpected

### ISSUE THREAD

**Issue by shelverr** [checkra1n/BugTracker#1025]
**Tell us about your setup:**
1. What iDevice are you using?
iPad Pro 9.7" 2016

2. On what version of iOS is it?
iOS 13.3

3. What version of checkra1n are you using? GUI or CLI?
0.9.8.2 - CLI

4. What is your host system (OS version? Hackintosh? VM? etc.)?
Raspberry Pi 4 revision 1.1

5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
From raspberry Pi using the stock lightning cable

**What are the steps to reproduce the issue?**
1.  wget https://assets.checkra.in/downloads/linux/cli/arm/c5cbb125c6948b39383702b62cec4f184263c8db50f49b9328013213126dae78/checkra1n
I also tried the ARM64 version but same issue

2. sudo chmod +x checkra1n

3. sudo ./checkra1n
...

**What do you expect, and what is happening instead?**
I expect it to go into checkra1n menu but it gives me an error

**Does the issue also occur if you enable Safe Mode?**

**Any other info, error logs, screenshots, ...?**

./checkra1n: Syntax error: redirection unexpected
just that

**Comment by Siguza** [comment]
It seems like it's trying to interpret checkra1n as a bash script. That sounds like your Pi installation is majorly messed up. o.0

**Comment by shelverr** [comment]
Weird, I did a clean install on a new micro SD and still the same issue

**Comment by bmart18** [comment]
I had the same issue on my VM running debian kali 64bit, im not sure if the issue was with the permission changing or the actual use of the file itself, however i switched over to my debian parrot 64 bit os and everything worked fine on there

**Comment by samkhazi** [comment]
same issue please solve this ./checkra1n: 3: Syntax error: redirection unexpected

**Comment by Siguza** [comment]
Well we need to figure out the steps to reproduce this first.  
Could you run `shasum -a 256 ./checkra1n` please?

**Comment by Potatopy** [comment]
I ran the command but it showed this

`bf5a0f30e208ec525617e1658dde27513fcca6e37ede1b6b361b7bae1ec7754b  ./checkra1n`

**Comment by Siguza** [comment]
Hmm, that is the right binary for arm64... could I get a `uname -a` too maybe?

**Comment by Potatopy** [comment]
Now this showed up
`uname: extra operand ‘256’`

**Comment by Siguza** [comment]
No, that was meant to be the full command. Just run `uname -a` with nothing else.

**Comment by Potatopy** [comment]
It just showed the Terminal details
`Linux ZacharyPC 4.4.0-19041-Microsoft #488-Microsoft Mon Sep 01 13:43:00 PST 2020 x86_64 x86_64 x86_64 GNU/Linux`

**Comment by Siguza** [comment]
So... you're on `x86_64`. Why are you trying to run an arm64 binary again?

**Comment by Potatopy** [comment]
Im on a A10 device with iOS 14.2

**Comment by Siguza** [comment]
No, you're on an x86_64 Linux installation. Use the x86_64 binary.

**Comment by Potatopy** [comment]
Right now im using the windows linux terminal because im too lazy to setup linux so maybe ill do it tomorrow
